test/components/bbox_polygon.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:test/test.dart';
+import 'package:turf/helpers.dart';
+import 'package:turf/src/bbox_polygon.dart';
+import 'package:turf/src/bbox.dart';
+
+main() {
+  test(
+    "bbox-polygon",
+    () {
+      var poly = bboxPolygon(Bbox(0, 0, 10, 10));
+      expect((poly.geometry is Polygon), equals(true));
+    },
+  );
+
+  test(
+    "bbox-polygon -- valid geojson",
+    () {
+      var poly = bboxPolygon(Bbox(0, 0, 10, 10));
+      var coordinates = poly.geometry!.coordinates;
+
+      expect((coordinates[0].length == 5), equals(true));
+      expect(coordinates[0][0][0] == coordinates[0][coordinates.length - 1][0],
+          equals(true));
+      expect(coordinates[0][0][1] == coordinates[0][coordinates.length - 1][1],
+          equals(true));
+    },
+  );
+
+  test(
+    "bbox-polygon -- Error handling",
+    () {
+      expect(() => bboxPolygon(Bbox(-110, 70, 5000, 50, 60, 3000)),
+          throwsA(isA<Exception>()));
+    },
+  );
+
+  test(
+    "bbox-polygon -- Translate BBox (Issue #1179)",
```

Comment:
I think, the issue reference is irrelevant for us here. WDYT?

Reply:
It depends on how we port the packages. I would remove it and leave it to the future users to check for the details.